### ISSUE TITLE
Crash when processing "font: unset" with level 2 optimizations enabled

### ISSUE THREAD

**Issue by nop00** [jakubpawlowicz/clean-css#940]
CleanCSS crashes when processing "font: unset" with level 2 optimizations enabled.  
Note that it does not crash with version 4.0.12 (crashes with versions 4.1.0 and 4.1.1).

### Environment

* clean-css version 4.1.1
* node.js version 6.10.3
* operating system: Windows 10

### Configuration options

```js
var CleanCSS = require('clean-css');
new CleanCSS({
    level: 2
})
```

### Input CSS

```css
.test {
  font: unset;
}
```

### Output

```
c:\clean-css-test\node_modules\clean-css\lib\optimizer\level-2\properties\populate-components.js:30
          throw e;
          ^

TypeError: Cannot read property '1' of undefined
    at Object.font [as breakUp] (c:\clean-css-test\node_modules\clean-css\lib\optimizer\level-2\break-up.js:308:66)
    at populateComponents (c:\clean-css-test\node_modules\clean-css\lib\optimizer\level-2\properties\populate-components.js:17:42)
    at optimizeProperties (c:\clean-css-test\node_modules\clean-css\lib\optimizer\level-2\properties\optimize.js:19:3)
    at recursivelyOptimizeProperties (c:\clean-css-test\node_modules\clean-css\lib\optimizer\level-2\optimize.js:63:9)
    at level2Optimize (c:\clean-css-test\node_modules\clean-css\lib\optimizer\level-2\optimize.js:77:3)
    at optimize (c:\clean-css-test\node_modules\clean-css\lib\clean.js:131:5)
    at c:\clean-css-test\node_modules\clean-css\lib\clean.js:104:29
    at c:\clean-css-test\node_modules\clean-css\lib\reader\read-sources.js:26:64
    at loadOriginalSources (c:\clean-css-test\node_modules\clean-css\lib\reader\load-original-sources.js:26:5)
    at c:\clean-css-test\node_modules\clean-css\lib\reader\read-sources.js:26:14
```

**Comment by jakubpawlowicz** [comment]
I can see it too - please use `{ level: { 2: { skipProperties: ['font'] } } }` until 4.1.2.

**Comment by nop00** [comment]
Thanks for the workaround !

**Comment by jakubpawlowicz** [comment]
The fix is out in 4.1.2.

**Comment by nop00** [comment]
Thanks!
